Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.6
+
+MAINTAINER Andy Driver <andy.driver@digital.justice.gov.uk>
+
+RUN apk update && \
+    apk add \
+        --virtual build-deps \
+        gcc \
+        musl-dev \
+        postgresql-dev \
```

Comment:
Is this needed to install some of the requirements? Postgres should run in a separate container right? (using docker compose to run the tests).